packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseOauthFlowsObject-test.js

```diff
@@ -0,0 +1,132 @@
+const { Fury } = require('fury');
+const { expect } = require('../../chai');
+const parse = require('../../../../lib/parser/oas/parseOauthFlowsObject');
+const Context = require('../../../../lib/context');
+
+const { minim: namespace } = new Fury();
+
+describe('Oauth Flows Object', () => {
```

Comment:
We're missing a lot of test coverage here, looks like there is no tests ensuring the success paths work that oauth flow is parsed as an instance of AuthScheme with grant types and scopes.

Reply:
That is because I am changing the spec for the success part of this. And I didn't want to rewrite tests. Therefore, I skipped them for now.